tests/cisco_asa/show_access-list/cisco_asa_show_access-list.yml

```diff
@@ -940,15 +970,17 @@ parsed_sample:
     entry_src_host: "10.1.1.10"
     entry_src_network: ""
     entry_src_mask: ""
+    entry_src_network6: ""
     entry_src_any: ""
     entry_src_fqdn_state: ""
     entry_dst_fqdn: ""
     entry_dst_range_start: ""
     entry_dst_range_end: ""
     entry_dst_host: ""
     entry_dst_network: ""
+    entry_dst_network6: "any4"
     entry_dst_mask: ""
-    entry_dst_any: "any4"
```

Comment:
Is this correct?

Reply:
I used the dev scripts to generate the yml verification file. I think the any4 is because i have added the traceroute icmp service too:

`access-list test line 34 extended permit icmp any4 169.254.151.0 255.255.0.0 traceroute (hitcnt=0) 0x480bef5c`

Reply:
I think @itdependsnetworks point is that this seems to break the parsing, as that should still be captured as `entry_dst_any`